apps/meteor/tests/end-to-end/api/28-roles.ts

```diff
@@ -140,4 +144,416 @@ describe('[Roles]', function () {
 				});
 		});
 	});
+
+	describe('[/roles.getUsersInRole]', () => {
+		let testUser: IUser;
+		let testUserCredentials: { 'X-Auth-Token': string; 'X-User-Id': string };
+		const testRoleName = `role.getUsersInRole.${Date.now()}`;
+		let testRoleId = '';
+
+		before(async () => {
+			await updatePermission('access-permissions', ['admin']);
+			testUser = await createUser();
+			testUserCredentials = await login(testUser.username, password);
```

Comment:
I think we need to create and change these after checking if it's enterprise right? Otherwise, the user won't be deleted.

Reply:
I need a test user to be created regardless of the license since it'll be used in the `should fail when user does NOT have the access-permissions permission` test case. But thanks for pointing that, I'll move this user deletion to before the `isEnterprise` check in `after`